test/distributed/test_c10d_common.py

```diff
@@ -321,10 +321,11 @@ def tearDown(self):
         # DistributedDataParallel test doesn't seem to call FileStore destructor
         # TODO: investigate this test and the test is known to have issues
         # Use this hack to remove files for that test
-        try:
-            os.remove(self.file_name)
-        except OSError:
-            pass
+        if hasattr(self, "file_name"):
+            try:
+                os.remove(self.file_name)
+            except OSError:
```

Comment:
nit, may be just use EAFP, `except (OSError, AttributeError):` and remove `if`?

Reply:
done -- didn't even think of that :)